mmv1/products/apigee/api.yaml

```diff
@@ -316,6 +316,31 @@ objects:
           - "PROGRAMMABLE"
           - "CONFIGURABLE"
         input: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'nodeConfig'
+        description: |
+          NodeConfig for setting the min/max number of nodes associated with the environment.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'minNodeCount'
+            description: |
+              The minimum total number of gateway nodes that the is reserved for all instances that
+              has the specified environment. If not specified, the default is determined by the
+              recommended minimum number of nodes for that gateway.
+            input: true
```

Comment:
If this is updatable, you do not want `input: true`, as this makes the field ForceNew